### ISSUE TITLE
NTR: AP-1R Adapter complex

### ISSUE THREAD

**Issue by suzialeksander** [geneontology/go-ontology#16947]
NTR: AP-1R Adapter complex
Synonym: AP-1 Related adapter complex

Similar to existing  GO:0030121 AP-1 adaptor complex, but CP has two different accessions.

"Yeast have two forms of AP-1, which differ only in the μ chain. Here we show that the variant μ chain Apm2 confers distinct cargo-sorting functions." (https://www.molbiolcell.org/doi/10.1091/mbc.e15-09-0621)

I'm not totally certain this warrants a NT, as it seems to technically fit the definition for GO:0030121 but this complex sorts a different cargo and is functionally distinct from AP-1.  I'm trying to annotate the variant μ chain (Apm2), which is not present in AP-1 and an annotation to AP-1 could be misleading. 

References:
PMID: 26658609 (same as above)
https://www.ebi.ac.uk/complexportal/complex/CPX-533

**Comment by bmeldal** [comment]
Hi Suzie,

I wonder if we should change the def of the existing term rather than adding a new one as their general function is the same (cargo sorting into the clathrin coat of forming vesicles). Once we make the human AP-1* complexes we'll have a bunch of variants and we won't make separate GO terms for these. The GO def already accounts for those variants, so why not for the yeast one? The 2 yeast complexes in the CP are both annotated to AP-1 (I wasn't the curator).

Birgit

**Comment by suzialeksander** [comment]
Yep, looking at some of the other complexes with varying parts, we have them all annotated to a general complex and users already have to sort through to figure out which pieces belong to the same whole- and later, there are terms like  GO:0035653 clathrin-coated vesicle cargo loading, AP-1-mediated that would be a pain do more NTRs for. 

So definitely adding AP-1R to synonyms would be great, no new term but massage the definition a little- that would be great, thanks

**Comment by bmeldal** [comment]
Glad we agree :)
Please you suggest the def changes, then an editor will commit them.

**Comment by ValWood** [comment]
Note that in go, all adaptors are spelled "adaptor".
Do these terms need synonyms for the 2 spellings of adaptor?

**Comment by bmeldal** [comment]
> Note that in GO, all adaptors are spelled "adaptor".
> Do these terms need synonyms for the 2 spellings of adaptor?

I googled it (as a non-native English speaker and someone with an interest in etymology these questions always intrigue me!) and essentially, adaptEr and adaptOr are used interchangably. But adaptEr seems more common theses days:
[https://writingexplained.org/adapter-vs-adaptor-difference](https://writingexplained.org/adapter-vs-adaptor-difference)

Although, some argue that if you refer to a person, e.g. a screen writer, it should be adaptOr as the suffix "-or" is more commonly used for persons (e.g. actor, as it comes from Latin).

In summary, go for one and add the other as EXACT synonyms everywhere.

**Comment by hdrabkin** [comment]
Could someone please state the exact proposed def for this term; I can add synonyms, etc.

**Comment by ValWood** [comment]
I also Googled it because until today I don't recall ever seeing "adapter" in a biological context so I was really surprised to see that this was more common usage...

**Comment by ValWood** [comment]
PubMed
adapter 7,707 
adaptor 56,709

Not quite a Thighland/Thailand ratio  but healthily in favour of adaptor

**Comment by hdrabkin** [comment]
using obo-edit to search just on name or synonym
adaptor: 100
adapter: 0
Synonym:
adaptor: 67
adapter: 2

interesting it's not used by go very much either. Adding 'adapter' synonym versions doesn't seem like critical.

**Comment by bmeldal** [comment]
WOW!!!
We are bucking the trend. Are we rebels?

**Comment by hdrabkin** [comment]
So how should I change the def if we don't need NTR so I can close?

**Comment by bmeldal** [comment]
@suzialeksander what should the new def be?

**Comment by pgaudet** [comment]
@suzialeksander ping

**Comment by suzialeksander** [comment]
It's been a while since I've looked at this.  I'm thinking that just adding 
 Synonym: AP-1 Related adapter complex

would work.  I can't decide if it's exact though. The difference between AP-1 and AP-1R, at least in yeast, is one subunit (which still fits with the original def), and the target isn't mentioned in the current definition.

**Comment by hdrabkin** [comment]
I can add it as related to close the ticket  and we can change it when you are sure.
